src/main/java/at/hannibal2/skyhanni/features/garden/FarmingMilestoneCommand.kt

```diff
@@ -0,0 +1,84 @@
+package at.hannibal2.skyhanni.features.garden
+
+import at.hannibal2.skyhanni.data.GardenCropMilestones
+import at.hannibal2.skyhanni.data.GardenCropMilestones.getCounter
+import at.hannibal2.skyhanni.features.garden.farming.GardenCropSpeed.getSpeed
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import at.hannibal2.skyhanni.utils.TimeUtils
+import net.minecraft.command.CommandBase
+
+object FarmingMilestoneCommand {
+
+    fun onCommand(crop: String?, current: String?, target: String?, needsTime: Boolean) {
+        if (crop == null) {
+            LorenzUtils.chat("No crop type entered")
+            return
+        }
+
+        val enteredCrop = CropType.entries.firstOrNull { it.simpleName == crop.lowercase() } ?: run {
+            LorenzUtils.chat("Invalid crop type entered")
+            return
+        }
+
+        val currentMilestone = getValidNumber(current)
+        val targetMilestone = getValidNumber(target)
+
+        if (currentMilestone == null) {
+            val currentProgress = enteredCrop.getCounter()
+            val currentCropMilestone = GardenCropMilestones.getTierForCropCount(currentProgress, enteredCrop) + 1
+            val output = (GardenCropMilestones.getCropsForTier(currentCropMilestone, enteredCrop) - currentProgress).formatOutput(needsTime, enteredCrop)
+
+            LorenzUtils.chat("§7$output needed for you to reach the next milestone")
+            return
+        }
+
+        if (targetMilestone == null) {
+            val output = GardenCropMilestones.getCropsForTier(currentMilestone, enteredCrop).formatOutput(needsTime, enteredCrop)
+
+            LorenzUtils.chat("§7$output needed for milestone §7$currentMilestone")
+            return
+        }
+        if (currentMilestone >= targetMilestone) {
```

Comment:
this should be >, not >=. and == should be its own label. or rephrase it as "x needs to be greater than y"